bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/StyledText.java

```diff
@@ -8751,6 +8749,13 @@ private boolean hasMultipleCarets() {
 	return carets != null && carets.length > 1;
 }
 
+/**
+ * See {@link TextLayout#setFixedLineMetrics}
+ */
+public void setFixedLineMetrics(FontMetrics metrics) {
```

Comment:
I believe with 99.9% probability this new method needs `@since 3.125` comment and it would be an API error if API tooling would work properly.

Reply:
Confirmed, it is needed.

Reply:
done